.github/configuration/dump-restore-test-configuration.yml

```diff
@@ -1,8 +1,20 @@
 dump-restore-version: [[
   {
-    version: source.latest
+    version: source.latest,
+    logical-database: null
   },
-  { 
-    version: target.latest
+  {
+    version: target.latest,
+    logical-database: null
+  }
+],
+[
+  {
+    version: source.latest,
+    logical-database: master
+  },
+  {
+    version: target.latest,
+    logical-database: master 
```

Comment:
For dump-restore, do we have different github actions for single-db and multi-db?

Reply:
It's okay for now, but when something is changed in the future, it might be hard to think about dump/restore cases.

Btw, I'm just wondering the following cases:
1. single-db --> multi-db
2. multi-db --> single-db
Do we allow those cases in a database-level dump/restore?

Reply:
Nope, as discussed earlier we only support the following:
1. single-db --> single-db
2. multi-db --> multi-db  

Added this in the description as well.